Essentials/src/com/earth2me/essentials/messaging/SimpleMessageRecipient.java

```diff
@@ -117,12 +117,11 @@ public MessageResponse onReceiveMessage(IMessageRecipient sender, String message
         if (!isReachable()) {
             return MessageResponse.UNREACHABLE;
         }
-        
+
         User user = getUser(this);
         boolean afk = false;
         if (user != null) {
-            if (user.isIgnoreMsg()
```

Comment:
Looks like you accidentally removed the `isIgnoreMsg` check - this still needs to be done or else users won't get messages.